# ChIP Seq Analysis Nanog and H3K27Ac

**Introduction** 


This repository contains work done from an independent project I undertook to perform a full ChIP-Seq analysis of the transcription factor Nanog on Zebrafish embryos using data from the [Sequence Read Archive (SRA)](https://www.ncbi.nlm.nih.gov/sra).

The 3 main questions I'm trying to answer:  

    - Is binding strength of Nanog different between promoter and enhancer peaks?

    - Are there different sequence motifs found in promoter peaks vs enhancer peaks?

    - Is binding strength and binding sequence motif of your transcription factor at enhancers
    correlated with H3K27Ac histone modification levels?


**Files** 

:scroll: `Supplementary Documentation.docx` 

Contains the specific commands used to perform ChIP-Seq analysis. This independent project utilizes High-Performance Computing cluster provided by University of Pittsburgh [Center for Research Computing (CRC)](https://crc.pitt.edu). 

:scroll: `nanog_heatmap.png` 

Preliminary results of comparing Nanog binding strength at promoters and enhancers. More work needs to be done in terms of analysis for the final publication written on this analysis. 

**Contact Information**

![My Interests](https://avatars1.githubusercontent.com/u/38919947?s=400&u=49ab1365a14fac78a91e425efd583f7a2bcb3e25&v=4)

Yogindra Raghav (YogiOnBioinformatics)

yraghav97@gmail.com
